### PR TITLE
Story/quad 86

### PR DESCRIPTION
**Pull request by prasanthpriyanesan**

### CODE REVIEW

File: Quadriga/.classpath [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+				<attribute name="org.eclipse.jst.component.dependency" value="/WEB-INF/lib"/>
+			</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
+			<attributes>
+				<attribute name="maven.pomderived" value="true"/>
+			</attributes>
+	</classpathentry>
+	<classpathentry kind="output" path="target/classes"/>
+</classpath>
```
Comment (jdamerow) [inline Quadriga/.classpath:30]: Don't commit the classpath file.

File: Quadriga/.settings/org.eclipse.wst.common.component [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+	    <wb-resource deploy-path="/" source-path="/target/m2e-wtp/web-resources"/>
+		   	   <wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
+	    <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
+	    <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/java"/>
+	    <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
+	    <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/resources"/>
+			   <property name="java-output-path" value="target/classes"/>
+	    <property name="context-root" value="quadriga"/>
+	   </wb-module>
+</project-modules>
```
Comment (jdamerow) [inline Quadriga/.settings/org.eclipse.wst.common.component:12]: don't commit settings file

File: Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/impl/workbench/PublicPageFactory.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	@Override
+	public IPublicPage clonePublicPageObject(IPublicPage publicpage) {
+		IPublicPage clone = new PublicPage();
+		clone.setTitle(publicpage.getTitle());
+		clone.setDescription(publicpage.getDescription());
+		clone.setOrder(publicpage.getOrder());
+		return clone;
+	}
+
+}
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/domain/factory/impl/workbench/PublicPageFactory.java:38]: please follow the style guide

File: Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+package edu.asu.spring.quadriga.domain.impl.workbench;
+
+/**
+ * @description : Project class describing the properties of a Public Page object
+ * 
+ * @author : Prasanth Priya Nesan
+ * 
+ */
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java:8]: this is not where javadoc goes

File: Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+public class PublicPage implements IPublicPage {
+    private String title;
+    private String description;
+    private int order;
+
+    /**
+     * retrieves the Title
+     */
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java:19]: please write proper javadoc

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    protected void initBinder(WebDataBinder binder) {
+        binder.setValidator(validator);
+    }
+
+    /**
+     * This method is called during the load of Public page settings form
+     * 
+     * @return model - model object
+     */
+    @PreAuthorize("hasRole('ROLE_QUADRIGA_USER_ADMIN')")
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java:48]: This is the role a Quadriga Admin user has, not the admin of a projects. Check out the access policy that is used in other controllers.

File: Quadriga/src/main/webapp/WEB-INF/views/workbench/addpublicpage.jsp [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+				<td style="width: 170px">Order Preference</td>
+				<td style="width: 1px"><form:input path="order" size="60"
+						id="order" /></td>
+				<td><form:errors path="order" class="ui-state-error-text"></form:errors></td>
+			</tr>
+			<tr>
+				<td><input type="submit" value="SAVE"></td>
+			</tr>
+		</table>
+	</form:form>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/addpublicpage.jsp:48]: Shouldn't there be three forms?

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java (hunk 6 above)
Comment (prasanthpriyanesan) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/PublicPageController.java:48]: I will check and change accordingly.

**Pull request closed without merging**